src/sectors.jl

```diff
@@ -220,6 +228,37 @@ Base.:&(::SimpleFusion, ::UniqueFusion) = SimpleFusion()
 Base.:&(::GenericFusion, ::UniqueFusion) = GenericFusion()
 Base.:&(::GenericFusion, ::SimpleFusion) = GenericFusion()
 
+# similar, but for multifusion categories
+"""
+    MultiFusionStyle(::Sector)
+    MultiFusionStyle(I::Type{<:Sector})
+
+Trait to describe the semisimplicity of the unit sector of type `I`.
+This can be either
+*   `SimpleMultiFusion()`: the unit is simple (e.g. fusion categories);
+*   `GenericMultiFusion()`: the unit is semisimple.
+"""
+abstract type MultiFusionStyle end
+MultiFusionStyle(a::Sector) = MultiFusionStyle(typeof(a))
+
+struct SimpleMultiFusion <: MultiFusionStyle end
+struct GenericMultiFusion <: MultiFusionStyle end
+
+MultiFusionStyle(::Type{<:Sector}) = SimpleMultiFusion() # default is simple unit
+
+# combine fusion properties of tensor products of multifusion sectors
```

Comment:
I think these overloads in the `FusionStyle` case are only really used for `ProductSector` instances, which I would expect require an overload here as well.